salt/acl/__init__.py

```diff
@@ -18,8 +18,9 @@ class PublisherACL(object):
     Represents the publisher ACL and provides methods
     to query the ACL for given operations
     '''
-    def __init__(self, blacklist):
+    def __init__(self, blacklist, whitelist):
```

Comment:
I'm not sure how this will impact any other calls being made to the PublisherACL. Glancing at the code it looks like the salt/master.py and salt/daemons/masterapi.py files are using it which are integral parts of salt. I would assume this would require us to make changes to those instances of this class as we are adding an arg here. Is it possible to add some tests without changing the salt code?

Reply:
@frogunder Did you see the comments from @Ch3LL above?